### DISCUSSION TITLE
How to override the default link color

### DISCUSSION THREAD

**Discussion by jmichelgarcia** [google/docsy#1585] [Q&A] [answered]
I initially though the way to define my own default link color was to set `$link-color` like I see it [here](https://github.com/google/docsy/blob/6f03b4bde7be06ae1f7b95c03a471bee0f57fbe7/assets/scss/_variables.scss#L37). However that doesn't seem to affect the link color.

I know I can override Bootstrap default variables by using something like this in my `_variables_...scss` file.

```
:root {
  --bs-link-color: #ff0000; /* Replace #ff0000 with your desired color value */
}
```

What would be the correct way to achieve this? Is it the above method the correct one? Sorry if this is a very basic BS question

![image](https://github.com/google/docsy/assets/2745523/1b4eac5d-cde8-45dd-9d37-505d43f3a3f5)

**Comment by jmichelgarcia** [comment]
@chalin, would you know by any chance?

**Comment by chalin** [comment]
Hi @jmichelgarcia - I'm catching up on notifications, since I was OOO last week. Have you resolved this and/or are your questions still pertinent?

**Reply by jmichelgarcia** [reply]
Still trying to get it to work.

I see the [variable exists](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss) in BS, but when I set it in `_variables_project.scss` it doesn't seem to work.

**Reply by jmichelgarcia** [reply]
Could it be due to [this](https://github.com/google/docsy/blob/841b9a0f38635488bad6b1617405b705cc39cb44/assets/scss/_variables.scss#L37) not having `!default` set?

**Reply by jmichelgarcia** [reply]
it used to be there, but it got [removed](https://github.com/google/docsy/commit/706dbf9af3d61338be9335e261d2a4538125800c)

**Reply by chalin** [reply, marked as answer]
Darn, yes! That should have `!default` -- apologies for that oversight! Here's the fix: #1631.
